### ISSUE TITLE
[BUG] logrotate error

### ISSUE THREAD

**Issue by johanburati** [Azure/WALinuxAgent#2374]
I have deployed a new RHEL VM connected to an OMS workplace, **logrotate** returns an error because of duplicate entries, in the **logrotate** config associated with **OmsAgentForLinux**  extension and the one associated with **WALinuxAgent**.

## Error
```
reading config file waagent-extn.logrotate
error: waagent-extn.logrotate:1 duplicate log entry for /var/log/azure/Microsoft.EnterpriseCloud.Monitoring.OmsAgentForLinux/extension.log
error: found error in file waagent-extn.logrotate, skipping
```
## Distro  
```
Red Hat Enterprise Linux release 8.4 (Ootpa)
```
## WALinuxAgent Version
```
WALinuxAgent-2.2.49.2 running on redhat 8.4
Python: 3.6.8
Goal state agent: 2.5.0.1
```

## OmsAgentForLinux Version
```
Version: 1.13.40
```

**Comment by johanburati** [comment]
Also open this issue https://github.com/microsoft/OMS-Agent-for-Linux/issues/1386

**Comment by narrieta** [comment]
Thanks for reporting this. Fixed by #2143, but the agent on the base image needs to be updated for the fix to propagate.
